tests/conftest.py

```diff
@@ -40,19 +32,6 @@
 hypothesis.settings.load_profile("ci")
 
 
-def set_evm_verbose_logging():
```

Comment:
hmm, it's helpful to be able to switch on debugging from the conftest.py (and it would be even more helpful if it turned on debugging in some fashion for any backend)

Reply:
Indeed both backends read this fixture:
https://github.com/vyperlang/vyper/blob/b0c3fa50bbda95aa16102f22d5ec76e8c9d0d3cb/tests/conftest.py#L126-L127